server/routes/teamAPI.js

```diff
@@ -63,11 +118,23 @@ router.post("/api/team/admin", auth, (req, res) => {
     .then(() => {
       db.User.updateOne({ _id: req.userId }, { $set: { isAdmin: false } });
```

Comment:
```suggestion
      db.User.updateOne({ _id: req.userId }, { $set: { isAdmin: false } }).then(() => {});
```
This seems to fix the changing user not being updated!